clang/www/make_cxx_dr_status

```diff
@@ -159,21 +165,37 @@ def availability(issue):
   if status == 'unknown':
     avail = 'Unknown'
     avail_style = ' class="unknown"'
-  elif re.match('^[0-9]+\.?[0-9]*', status):
-    avail = 'Clang %s' % status
-    if float(status) > latest_release:
-      avail_style = ' class="unreleased"'
-    else:
-      avail_style = ' class="full"'
+  elif re.match(r'^[0-9]+\.?[0-9]*', status):
+    if not proposed_resolution:
+      avail = 'Clang %s' % status
+      if float(status) > latest_release:
+        avail_style = ' class="unreleased"'
+      else:
+        avail_style = ' class="full"'
+    else: 
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang {status} implements {proposed_resolution} resolution"'
   elif status == 'yes':
-    avail = 'Yes'
-    avail_style = ' class="full"'
+    if not proposed_resolution:
+      avail = 'Yes'
+      avail_style = ' class="full"'
+    else:
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang implements {proposed_resolution} resolution"'
   elif status == 'partial':
-    avail = 'Partial'
-    avail_style = ' class="partial"'
+    if not proposed_resolution:
+      avail = 'Partial'
+      avail_style = ' class="partial"'
+    else:
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang partially implements {proposed_resolution} resolution"'
   elif status == 'no':
-    avail = 'No'
-    avail_style = ' class="none"'
+    if not proposed_resolution:
+      avail = 'No'
+      avail_style = ' class="none"'
+    else:
+      avail = 'Not Resolved*'
+      avail_style = f' title="Clang does not implement {proposed_resolution} resolution"'
```

Comment:
In the no case, I would leave it unknown / no comment... that we don't do something is not useful information for users. WDYT?

Reply:
I tend to agree; if we're telling users "there's some drafting and we do what the drafting said as of XXX", then there's similar value in telling users "there's some drafting and we do not do what the drafting said as of XXX" (for example, as the issue is resolved, it tells us that we can convert it into a No instead of an Unknown).

Reply:
I don't agree (because telling users we don;t do something does not tell them what we do), but I will bow to whatever decision @AaronBallman takes.

Reply:
I think we should leave it. I agree that it doesn't tell users what we're actually doing, but the point here is more for a user who reads the draft wording and says "Clang isn't doing what that says, is that a bug?" without really realizing that "drafting" status may mean we don't want to work on the issue until a final decision has been reached. If we explicitly say "we don't do what the current drafting says", it at least tells the user "we're aware of the issue, don't expect us to do what the current drafting says".